cellengine/utils/api_client/APIClient.py

```diff
@@ -207,10 +207,9 @@ def post_experiment(self, experiment: dict, as_dict=False) -> Experiment:
             return experiment
         return Experiment.from_dict(experiment)
 
-    def clone_experiment(self, _id, name=None, as_dict=False) -> Experiment:
-        experiment = self._post(
-            f"{self.base_url}/experiments/{_id}/clone", json={"name": name}
-        )
+    def clone_experiment(self, _id, name: str = None, as_dict=False) -> Experiment:
```

Comment:
This API also accepts `path: List[str]`. Maybe go ahead and change the `name` arg to a `props` dict. I imagine that will be more consistent between various functions in the toolkit.

Reply:
I mostly agree, although sometimes having a named arg is more obvious for the user than understanding what the available keys are for a `props` dict.  I'll open an issue to make this consistent

Reply:
https://github.com/primitybio/cellengine-python-toolkit/issues/139

Reply:
You know, you're right. Sorry to flip-flip here. Python's named args make them about as easy to use as a dict, and named args are probably easier to document. The GCS APIs take a big ol' list of args too: https://googleapis.dev/python/storage/latest/buckets.html#google.cloud.storage.bucket.Bucket.copy_blob. Change back? sorry again.